packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/CozyMarkerBuilder.java

```diff
@@ -175,4 +200,14 @@ public Bitmap buildMarker(String type, String text) {
         }
     }
 
+    public Bitmap buildMarker(String type, String text) {
+        String key = String.format("%s:%s", type, text);
+        final Bitmap bitmap = getBitmapFromMemCache(key);
+        if (bitmap != null) {
+            return bitmap.copy(bitmap.getConfig(), true);
```

Comment:
WDY about create a method to maybe make this bitmap.copy part more clear about what it's happening?